### PR TITLE
📖  More detail on difference between amp-subscriptions and amp-access

### PR DESCRIPTION
**Pull request by ed-bird**
- Adding more guidance on how to migrate from amp-access to amp-subscriptions
- Change image rendering so it loads on amp.dev

### CODE REVIEW

File: extensions/amp-subscriptions/amp-subscriptions.md [hunk 1]
```diff
@@ -46,14 +46,16 @@ The `amp-subscriptions` component is similar to [`amp-access`](../amp-access/amp
 and in many features builds on top of `amp-access`. However, it's a much more
 specialized version of access/paywall protocol. Some of the key differences are:
 
-1. The `amp-subscriptions` entitlements response is similar to the amp-access
-   authorization, but it's strictly defined and standardized.
-2. The `amp-subscriptions` component allows multiple services to be configured
+1. The `amp-subscriptions` [authorization endpoint](#authorization-endpoint) is similar to the
+   [`amp-access` authorization endpoint](../amp-access/amp-access.md#authorization-endpoint) but it's response is strictly defined and standardized.
```
Comment (ChrisAntaki) [inline extensions/amp-subscriptions/amp-subscriptions.md:50]:
```suggestion
   [`amp-access` authorization endpoint](../amp-access/amp-access.md#authorization-endpoint) but its response is strictly defined and standardized.
```

**Review by ChrisAntaki — APPROVED** [review]

**Comment by ChrisAntaki** [comment]
Hey @jpettitt ! Could you review this migration advice? I'm not so familiar with `amp-access`

File: extensions/amp-subscriptions/amp-subscriptions.md [hunk 2]
```diff
@@ -413,7 +415,11 @@ By default, as the body, pingback POST request receives the entitlement object r
...
 To accurately identify the Reader, the Publisher should associate the [AMP Reader ID][1] with any Publisher cookies relevant to the Reader.
 
-![Process Diagram](images/reader-id-assoociation.png)
+<amp-img alt="reader id cookie association" layout="responsive" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-subscriptions/images/reader-id-assoociation.png">
+  <noscript>
+    <img alt="reader id cookie association" layout="responsive" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-subscriptions/images/reader-id-assoociation.png">
+  </noscript>
+</amp-img>
 
 **Note:** due to the way that the [AMP Reader ID][1] is created, there may be multiple [AMP Reader IDs][1] for the same the Reader so the Publisher should be able to handle that appropriately.
```
Comment (jpettitt) [inline extensions/amp-subscriptions/amp-subscriptions.md:424]:
```suggestion
**Note:** due to the way that the [AMP Reader ID][1] is created, there may be multiple [AMP Reader IDs][1] for the same the user across different devices and browsers so the Publisher must take care to handle that appropriately.
```

**Review by jpettitt — APPROVED** [review]
LGTM once @ChrisAntaki suggestions incorporated. My suggested change is an optional clarification nit.

File: extensions/amp-subscriptions/amp-subscriptions.md (hunk 2 above)
Comment (ChrisAntaki) [reply extensions/amp-subscriptions/amp-subscriptions.md:424]: Thanks for the quick review!

**Comment by ed-bird** [comment]
Thanks both!

**Comment by ChrisAntaki** [comment]
I noticed the AMP check is failing. They want markdown files formatted with prettier. Could you try running prettier on the markdown file? I think it's a matter of moving/deleting spaces

**Comment by ed-bird** [comment]
Should be good now, PTAL :)

**Comment by ChrisAntaki** [comment]
Good stuff, merged!

**Pull request merged**